docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
I like `daemon` too. Did you also consider `pipe` (takes a stream, returns a stream)?